query_optimizer/rules/UnnestSubqueries.hpp

```diff
@@ -180,6 +181,7 @@ class UnnestSubqueriesForExpession : public Rule<expressions::Expression> {
       const expressions::ExpressionPtr &node);
 
   void transformExists(const expressions::Exists &exists_predicate);
+  void transformInTableQuery(const expressions::InTableQuery &in_table_query);
```

Comment:
Please add an empty line between lines `183` and `184` to separate two methods. Also, add `doxygen` comments for them. Thanks!